roadmap.md

```diff
@@ -0,0 +1,54 @@
+# WebDriver BiDi roadmap
+
+## Real-world end-to-end user scenarios
+
+This document presents an overview of real-world end-to-end user scenarios we aim to enable via the WebDriver BiDi protocol. Each scenario requires one or more WebDriver BiDi commands and events to be specified, tested, and implemented across browser engines.
+
+### A. Extracting content
+
+This scenario loads a web page and uses script evaluation to extract content on the page (e.g. the headlines). In spec terms, this involves:
+
+- [x] [handling sessions](https://w3c.github.io/webdriver-bidi/#module-session)
+- [x] [creating a new `browsingContext`](https://w3c.github.io/webdriver-bidi/#command-browsingContext-create)
+- [x] [navigating to a URL](https://w3c.github.io/webdriver-bidi/#command-browsingContext-navigate)
+- [x] [evaluating JavaScript in the page context](https://w3c.github.io/webdriver-bidi/#command-script-evaluate)
+- [x] [serialization/deserialization of JavaScript values](https://w3c.github.io/webdriver-bidi/#data-types-protocolValue)
+- [ ] [sandboxed script execution](https://github.com/w3c/webdriver-bidi/issues/144)
+
+### B. Submitting forms
+
+This scenario loads a web page, enter text into a form field via the keyboard, and submits the form via a mouse click before extracting the results from the page. In spec terms, this involves:
+
+- [x] (everything from the previous milestone)
+- [ ] [emulating keyboard input](https://github.com/w3c/webdriver-bidi/pull/175)
```

Comment:
In practice the plan is to just port the actions API to BiDi directly, so we should get all the input types at once. However I understand that from an implementation point of view it might not be that simple, and that mouse/keyboard are the lowest hanging fruit here.

Reply:
Ack.

Reply:
Ack. Did you want me to change anything here, or did you just mean to provide that context?